spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/mapping/SpannerPersistentEntityImpl.java

```diff
@@ -91,18 +91,6 @@
 
   private final Set<Class<?>> jsonProperties = new HashSet<>();
 
-  /**
-   * Creates a {@link SpannerPersistentEntityImpl}.
-   *
-   * @param information type information about the underlying entity type.
-   */
-  public SpannerPersistentEntityImpl(TypeInformation<T> information) {
```

Comment:
It's a public method, unfortunately, so deprecate it instead. We can remove it in the next major release.